lib/chef/knife/config_list.rb

```diff
@@ -21,9 +21,8 @@ class Chef
   class Knife
     class ConfigList < Knife
       banner "knife config list (options)"
-      category "deprecated"
 
-      TABLE_HEADER = [" Profile", "Client", "Key", "Server"].freeze
+      TABLE_HEADER = [" Profile", "Client", "Key", "Server"].freeze unless const_defined?(:TABLE_HEADER)
```

Comment:
Getting already initialized warning suppress it by checking it's already defined or not.

Reply:
We really need to track down why this happens, but the easiest way to get around it is ||= instead of =

Reply:
seems it's an issue with all knife commands, used `||=` for now. Thanks!